sections/05_treat.md

```diff
@@ -146,7 +153,9 @@ information from protein-protein or protein-compound interaction networks
 [@doi:10.1186/1758-2946-5-30 @doi:10.1021/ci500340n
 @doi:10.1186/s12859-015-0845-0].
 
-Menden et al [@doi:10.1371/journal.pone.0061318] used a shallow neural network
+`TODO: shallow NN? Is that in scope?`
```

Comment:
@agapow I plan to remove the TODO about the shallow network and then work on cleaning up the conflicts and merging.  If you still prefer removing the shallow network, let's continue the discussion in a new issue or pull request.  I'd like to merge your treat changes with mine before we diverge too much.